client/src/App.tsx

```diff
@@ -1,33 +1,17 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import { Routes, Route } from 'react-router-dom'
+import CreateJob from './pages/CreateJob'
```

Comment:
I think we want to call it a task instead of a job?

Reply:
Yes, task is better, it's done